src/main/java/module-info.java

```diff
@@ -0,0 +1,3 @@
+module breeze {
+    requires jdk.incubator.vector;
```

Comment:
If a modularized application sees our module it will see no packages or classes as it is. We need to export any packages that we want to be open to the rest of the world. Maybe this should be its own issue? From what I can see there are no usage of the incubator vector api yet? So it isn't needed at all right now?